### ISSUE TITLE
Advanced permissions hard to read

### ISSUE THREAD

**Issue by schiessle** [nextcloud/groupfolders#938]
The sidebar doesn't have enough space for the advanced permissions. You can hardly read the group name and there is no way to distinguish multiple groups if the start with the same letters.

![image](https://user-images.githubusercontent.com/1589737/83871170-e1187780-a72f-11ea-97f9-f19ef10bfc2c.png)

A hover effect which shows the whole group name and column description might help. Or, but this would be a more general change, if the user could resize the sidebar.

Further the option at the end of the row to delete the rule again is cut of and not reachable.

**Comment by hokascha** [comment]
until further decision, you can add the following to your theme's server.css:
```
#app-sidebar{
        width:unset!important;
        max-width: unset!important;
}
```
This disables maximum width rules for the sidebar.

**Comment by fschrempf** [comment]
+1 for a mouseover/hover description for the group name

**Comment by ischiros** [comment]
I face the same problem, and confirm that the full name on hover would be greatly appreciated.

**Comment by fschrempf** [comment]
This has been implemented by 85c1866d78ecb9b296fb3758d742c2fdc518488c and is available in the latest release. Thanks @danxuliu! :tada:
